src/core/trulens/core/tru.py

```diff
@@ -26,12 +26,15 @@
 )
 
 import pandas
+import pandas as pd
```

Comment:
Theere is already an import on pandas right above, why not just change that to pd?

Reply:
oops - fixed now